Mathlib/Topology/AlexandrovDiscrete.lean

```diff
@@ -51,3 +54,108 @@ lemma isClopen_sUnion (hS : ∀ s ∈ S, IsClopen s) : IsClopen (⋃₀ S) :=
 
 lemma isClopen_iUnion (hf : ∀ i, IsClopen (f i)) : IsClopen (⋃ i, f i) :=
 ⟨isOpen_iUnion λ i ↦ (hf i).1, isClosed_iUnion λ i ↦ (hf i).2⟩
+
+lemma interior_iInter (f : ι → Set α) : interior (⋂ i, f i) = ⋂ i, interior (f i) :=
+(interior_maximal (iInter_mono λ _ ↦ interior_subset) $ isOpen_iInter λ _ ↦
+  isOpen_interior).antisymm' $ subset_iInter λ _ ↦ interior_mono $ iInter_subset _ _
```

Comment:
I'm pretty sure that our new convention is to indent the body of all declarations, even in term mode. Also, `<|` is preferred to `$`, and `fun` to `λ`.

Reply:
How is `<|` preferred over `$`? This does not appear in the [style guide](https://leanprover-community.github.io/contribute/style.html).

Reply:
I'm not looking for style suggestions, really. Especially when they involve more ASCII art than the alternative.

Reply:
I'm unresolving this at least for the indentation part, which is clearly stated in the style guide.

Reply:
Regarding "mathportism", I really don't think that's what's happening here. People have already spent quite some time doing new maths in Lean4, and there's been complains about a lot of things but almost none about this. I can't speak for everyone, but at least I willingly embraced the new style for the most part, and not because it was used by mathport.

Of course that doesn't mean that it's clearly better, but I would say that for now it's better to stick to what's used in the existing *human-written* codebase, and that means `fun` and `<|`.

Reply:
Then your argument is saying that I can stick to `λ` and `$`: `λ` is used 395 times across 79 files, and `$` at least 619 times across 202 files.

Again, I am not looking for style suggestions.